mnemonic/src/Cardano/Mnemonic/Util.hs

```diff
@@ -0,0 +1,79 @@
+-- | Module for small utility functions.
```

Comment:
I would say this (duplicate) `Util` module should be dropped, and the `eitherToParser` function duplicated to `BIP39.hs` — rather than putting it in a shared library.